src/Psl/Xml/format_error.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Xml;
+
+use LibXMLError;
+use Psl\Str;
+
+/**
+ * This function can be used to format a LibXMLError.
+ *
+ * @psalm-suppress MissingThrowsDocblock
+ */
+function format_error(LibXMLError $error): string
+{
+    return sprintf(
```

Comment:
```suggestion
    return Str\format(
```

we should avoid using php std functions if we have an alternative 😉